src/routes/users.js

```diff
@@ -1,27 +1,13 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-console */
-import express from 'express';
-import bcrypt from 'bcrypt';
-import { User } from '../database/models';
+import express from "express";
+import  * as User from '../controllers/userController';
 
-const userRoute = express.Router();
-
-userRoute.post('/new', async (req, res) => {
-  const { fName, lName, email, password } = req.body;
-  const salt = await bcrypt.genSalt(10);
-  const hashedPassword = await bcrypt.hash(req.body.password, salt);
-  try {
-    await User.create({
-      fName,
-      lName,
-      email,
-      password: hashedPassword
-    });
-    return res.send(201, 'User Registered');
-  } catch (error) {
-    console.log({ Error: error });
-    return res.status(500).json(error);
-  }
+const userRouter=express.Router();
+userRouter.post('/user/register',(req,res,next)=>{
+    try {
+        User.default.registerNew(req.body,res);
```

Comment:
Calling the controller "User" can be confusing since the model is also called "User"